AsyncDisplayKit/ASCollectionData.h

```diff
@@ -0,0 +1,131 @@
+//
+//  ASCollectionItem.h
+//  AsyncDisplayKit
+//
+//  Created by Adlai Holler on 11/4/16.
+//  Copyright © 2016 Facebook. All rights reserved.
+//
+
+#pragma once
+#import <Foundation/Foundation.h>
+#import <AsyncDisplayKit/ASDimension.h>
+
+@class ASCellNode;
+
+NS_ASSUME_NONNULL_BEGIN
+
+typedef NSString * ASSectionIdentifier;
+typedef NSString * ASItemIdentifier;
+typedef NSString * ASSupplementaryElementKind;
```

Comment:
Is it important these are strings, or could they be NSCopying or some other more lenient protocol?  Strings are fine if needed, just thinking there could be incremental value in it being more open if it would be supported easily.